account_financial_report_qweb/report/trial_balance.py

```diff
@@ -75,7 +75,7 @@ class TrialBalanceReportAccount(models.TransientModel):
     )
     hide_line = fields.Boolean(compute='_compute_hide_line')
     # Data fields, used to keep link with real object
-    sequence = fields.Integer(index=True, default=1)
+    sequence = fields.Char(index=True, )
```

Comment:
Redundant comma. Also, in v12 it has `default='1'` here. Personally can't see when would sequence not get set, but maybe still leave it?
```suggestion
    sequence = fields.Char(index=True)
```